Makefile

```diff
@@ -52,4 +53,5 @@ install: primus-os.bin
 	sudo cp $< /boot/primus-os.bin
 
 clean:
-	rm -f *.o primus-os primus-os.iso primus-os.bin $(OBJ_DIR)/*.o
\ No newline at end of file
+	rm -f *.o primus-os primus-os.iso primus-os.bin $(OBJ_DIR)/*.o
```

Comment:
You can include it in a single line, like so:
`rm -rf *.o primus-os primus-os.iso primus-os.bin $(OBJ_DIR)/*.o iso`